stubs/cachetools/cachetools/cache.pyi

```diff
@@ -1,20 +1,22 @@
-from typing import Callable, Generic, Iterator, Optional, TypeVar
+from typing import Callable, Generic, Iterator, Optional, TypeVar, Union
 
 from .abc import DefaultMapping as DefaultMapping
 
+NumericType = Union[int, float]
```

Comment:
Consider naming this `_NumericType`, so that this won't show up in autocompletions.

Reply:
Actually, you can just use `float`, because as far as type checkers are concerned, any `int` is also a valid `float`.